firefox-ios/Client/Frontend/Microsurvey/Survey/View/MicrosurveyViewController.swift

```diff
@@ -223,6 +223,7 @@ final class MicrosurveyViewController: UIViewController,
         scrollContainer.addArrangedSubview(containerView)
         scrollContainer.addArrangedSubview(submitButton)
         scrollContainer.addArrangedSubview(privacyPolicyButton)
+        scrollContainer.accessibilityElements = [containerView, submitButton, privacyPolicyButton]
```

Comment:
Only minor concern here offhand is whether this has the potential to override any other system accessibility elements that would potentially also be in the scroll container? Otherwise LGTM

Reply:
@mattreaganmozilla thanks for expressing the concern, its a good one. The system isn't correctly determining the ordering of the elements in medium detent only. This issue is resolved if the user expands and then returns to medium detent. As a workaround, I decided to enforce the ordering for the scroll container. After testing with voice over, it seems that the ordering is expected and I see no other issues. 

With that, I would lean towards a11y testing and reporting any bugs if there are issues in the future. This will be contained to the `scrollContainer` for now.

Reply:
> As a workaround, I decided to enforce the ordering for the scroll container. 

Yeah that makes sense thanks @cyndichin , just to clarify: my concern was just whether we should be overriding `accessibilityElements` completely with this collection literal, or querying the property and reordering the collection of elements, and then re-setting it to the same property. (I honestly am not sure if it matters offhand, but I think one has the potential risk of removing other a11y elements that may have been unexpectedly added by other code or UIKit, and the other doesn't.) But if VO is working this seems fine to me.

Reply:
Thanks for clarifying! Will keep an eye out on this. It seems we are overriding in multiple places so I'll follow the current convention for now. Appreciate you raising the concern! I'll also check in with Winnie.